### PR TITLE
initial lesson upload

### PR DESCRIPTION
**Pull request by thejammerr**

### CODE REVIEW

File: content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Inscribed and Central Angles
+#### Richard Hanxu
+
+A first look at competitive geometry, with an introduction to inscribed and central angles in a circle.
+Inscribed and Central Angle formulae are integral components to circle geometry problems. It's very likely that you'll encounter a problem that uses this concept. Know it well!
+
+<hr>
+
+<div class="embed-container">
```
Comment (0ffz) [inline content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md:14]: Should use `{% include youtubePlayer.html id="JIXOlER7qXU" %}` instead. Same goes for any of the other videos.

**Review by 0ffz — CHANGES_REQUESTED** [review]

File: content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md (hunk 1 above)
Comment (thejammerr) [reply content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md:14]: where exactly?

Comment (0ffz) [reply content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md:14]: Just replace the entire div container for all the pages where u used it

File: content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Inscribed and Central Angles
```
Comment (0ffz) [inline content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md:3]:
```suggestion
title: Inscribed and Central Angles
author: Richard Hanxu
```

File: content/math/algebra/geometry/circle-geometry/thales-theorem.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Thales Theorem
+---
+
+
+# Thales Theorem
+#### Richard Hanxu
+This is a brief extension of the Inscribed Angle theorem, which is useful in questions involving right triangles inscribed within a circle.
+<br>
```
Comment (0ffz) [inline content/math/algebra/geometry/circle-geometry/thales-theorem.md:10]: No need for a line break, two enters does the job.

File: content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# Inscribed and Central Angles
```
Comment (0ffz) [inline content/math/algebra/geometry/circle-geometry/inscribed-central-angles.md:6]: No need to repeat the lesson name, the `title` tag will put it on the page anyways.
```suggestion
```

**Review by 0ffz — CHANGES_REQUESTED** [review]
Just some more stuff I noticed, other than that, it looks good

**Pull request merged**